industry_benchmarks/utils/fix_networks.py

```diff
@@ -144,13 +145,19 @@ def _check_and_deduplicate_transforms(
     for t_name, t_list in transforms_dict.items():
         if len(t_list) != 3:
             # TODO: we can turn this into a warning message if it's too painful
-            errmsg = (
-                f"Too few transformations found for {t_name} "
-                "this indicates a partially completed set of results. "
-                "Please ensure that your input network is finished and "
-                "any reproducible partial failures have been removed."
-            )
-            raise ValueError(errmsg)
+            if allow_missing:
```

Comment:
The way it is written right now would allow missing results, e.g. if 2 out of the 3 solvent transformations failed. However, if three solvent transformations completed successfully and three complex simulations failed, this script will automatically remove the edge, even when the `allow_missing` flag is not set. Should we change this?